src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: getWidth();
+  padding: 100px $min-padding;
+
+  @each $weekDay, $dayNumber in $weekDays {
+    &--first-day--#{$weekDay} &__day:first-child {
+      margin-left: getMargin($dayNumber);
+    }
+  }
+
+  @for $month-end from 28 through 31 {
+    &--month-length--#{$month-end} &__day:nth-child(n+#{$month-end + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $dayBlock-size;
+    height: $dayBlock-size;
+
+    font-size: 30px;
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    transition: transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Move this parameter to the general selector.
You should write in hover only those parameters which are changing

Reply:
thank you! fixed :)